beetmoverscript/maven.py

```diff
@@ -0,0 +1,54 @@
+import os
```

Comment:
I'm tempted to see this file renamed to `maven_utils.py`. Not sure if it's worth it though. What say you?

Reply:
Fine by me. Done in https://github.com/mozilla-releng/beetmoverscript/pull/163/commits/5cab293b61869aaba9bfdb5c5ad756467a09bccd